scram-tools.file/tools/tensorflow-xla-runtime/tensorflow-xla-runtime.xml

```diff
@@ -0,0 +1,10 @@
+<tool name="tensorflow-xla-runtime" version="@TOOL_VERSION@">
+  <client>
+    <environment name="TENSORFLOW_XLA_RUNTIME_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR" default="$TENSORFLOW_XLA_RUNTIME_BASE/lib/archive"/>
+    <environment name="INCLUDE" default="$TENSORFLOW_XLA_RUNTIME_BASE/include"/>
```

Comment:
@riga, I guess this is not needed any more ... right?

Reply:
Right. It's removed now.